cyclonedx/model/__init__.py

```diff
@@ -147,13 +226,43 @@ class ExternalReferenceType(Enum):
     LICENSE = 'license'
     MAILING_LIST = 'mailing-list'
     OTHER = 'other'
+    RELEASE_NOTES = 'release-notes'  # Only supported in >= 1.4
```

Comment:
:100:  note for mysel: see of 1.3 serializers ommit this

Reply:
They don't at present @jkowalleck. Raised #133 to track.